src/components/FilterByCategory/FilterByCategory.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+
+const FilterByCategory = ({ renderProductsByCategories, categories }) => {
+  return (
+    <Box sx={{ width: "100%", maxWidth: 360, bgcolor: "background.paper" }}>
```

Comment:
Aquí podrias guardar estos estilos en una variable y solo pasarla para que sea mas fácil entender todo. 
```suggestion
   const boxStyle = { width: "100%", maxWidth: 360, bgcolor: "background.paper" };
    <Box sx={ boxStyle }>
```